### ISSUE TITLE
unable to load GateOne in DSM7.2

### ISSUE THREAD

**Issue by RobG-BW** [vletroye/SynoPackages#69]
when i try to load MODS-GateOne as a manual spk, it fails with 
"Invalid file format. please contact the package developer"

running DSM7.2.1-69057   , loading the DSM7 spk

and your SPK source is not recognised as valid.

**Comment by vletroye** [comment]
Indeed, (I started to adapt my packages to support DSM 7.2, but there are so many limitations that I stopped).
GateOne only run fine with elevated rights 

I should delete the sources from GitHub (I didn't even remember I pushed them here). Sorry.

**Comment by RobG-BW** [comment]
understandable. thanks for update.
